scripts/array.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+myArray=(1 2 3 manju ganesh 3333)
+
+echo "first element of an array: ${myArray[0]}"
+
+echo "All elements of an array: ${myArray[@]}"
```

Shellcheck

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)